Source/Scene/QuadtreePrimitive.js

```diff
@@ -128,12 +142,44 @@ define([
          */
         this.tileCacheSize = defaultValue(options.tileCacheSize, 100);
 
+        /**
+         * Gets or sets the number of loading descendant tiles that is considered "too many".
+         * If a tile has too many loading descendants, that tile will be loaded and rendered before any of
+         * its descendants are loaded and rendered. This means more feedback for the user that something
+         * is happening at the cost of a longer overall load time. Setting this to 0 will cause each
+         * tile level to be loaded successively, significantly increasing load time. Setting it to a large
+         * number (e.g. 1000) will minimize the number of tiles that are loaded but tend to make
+         * detail appear all at once after a long wait.
+         * @type {Number}
+         * @default 20
+         */
+        this.loadingDescendantLimit = 20;
+
+        /**
+         * Gets or sets a value indicating whether the ancestors of rendered tiles should be preloaded.
+         * Setting this to true optimizes the zoom-out experience and provides more detail in
+         * newly-exposed areas when panning. The down side is that it requires loading more tiles.
+         * @type {Boolean}
+         * @default true
+         */
+        this.preloadAncestors = true;
+
```

Comment:
This and `preloadSiblings` seem like pretty useful and easy to understand parameters, but there's currently no way to modify these from the public API right? (I know the development/Terrain Tweaks Sandcastle shows toggling them). I think these might be worth exposing, maybe on `Globe.js`, and adding them as checkboxes to the `Terrain.html` Sandcastle.

Reply:
I added these properties to `Globe`. Added `loadingDescendantLimit` and `fillHighlightColor` too. I'm a little hesitant to add them to `Terrain.html` though, cause I think they'll just make terrain look overcomplicated there.